finality-provider/config/config.go

```diff
@@ -26,17 +26,23 @@ const (
 	DefaultRPCPort                     = 12581
 	defaultConfigFileName              = "fpd.conf"
 	defaultNumPubRand                  = 50000 // support running of roughly 5 days with block production time as 10s
-	defaultNumPubRandMax               = 500000
-	defaultTimestampingDelayBlocks     = 6000 // 100 BTC blocks * 600s / 10s
-	defaultBatchSubmissionSize         = 1000
-	defaultRandomInterval              = 30 * time.Second
+	defaultBatchSubmissionSize         = 100
 	defaultSubmitRetryInterval         = 1 * time.Second
 	defaultSignatureSubmissionInterval = 1 * time.Second
 	defaultMaxSubmissionRetries        = 20
 	defaultBitcoinNetwork              = "signet"
 	defaultDataDirname                 = "data"
 )
 
+// Constants for system parameters validation limits
+const (
+	MaxBatchSize            = 100              // Maximum allowed batch submission size
+	MaxPubRand              = 500000           // Maximum allowed public randomness number
+	MinPubRand              = 8192             // Minimum allowed public randomness number
+	TimestampingDelayBlocks = 18000            // 300 BTC blocks * 600s / 10s where 300 BTC blocks is the system parameter of BTC block time required by the btc timestamping protocol
```

Comment:
What's the reasoning behind the `600s / 10s` part?

Reply:
10s is the babylon block production time. TimestampingDelayBlocks basically converts `w` into babylon blocks

Reply:
How about we increase this to account for some buffer? I wouldn't even mind doubling it.

Reply:
Increasing it can reduce the number of wasted randomness due to timestamping but also delay the time of an fp becoming active. Given that the fp will have 2 available commits in the air, I think it's fine. Maybe can increase it a little bit like `20000`?